harness/determined/pytorch/_callback.py

```diff
@@ -85,28 +91,38 @@ def load_state_dict(self, state_dict: Dict[str, Any]) -> None:
         pass
 
 
-# TODO(DET-3267): deprecate this when releasing pytorch flexible primitives.
+# TODO(DET-3262): remove this backward compatibility of old interface.
 class ClipGradsL2Norm(PyTorchCallback):
     """
-    Callback that performs gradient clipping using
+    (Deprecated) Callback that performs gradient clipping using
```

Comment:
Can we use `.. warning::` syntax for deprecation warnings rather than `(Deprecated)`?